src/main/resources/story/reddit/CreateAccountReddit.story

```diff
@@ -0,0 +1,33 @@
+Description: Account creation
+
+Scenario: Open reddit.com main page
+Meta:
+   @test
+Given I am on the main application page
+
+Scenario: Create user account
+Meta:
+   @test
+When I refresh the page
```

Comment:
Is this step required?

Reply:
@apaulenkova when the main page opens, appears a request to send notifications.  The best workaround, I thought that moment, was to refresh the page. I couldn't even inspect this element.